docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,161 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
```

Comment:
Please start this documentation page with a short explanation what is `HasMany` relation. I think we can pretty much copy the "Overview" section from https://loopback.io/doc/en/lb3/HasMany-relations.html

Reply:
✅
 _(this has been resolved by content added below the line 10)_